src/features/auth/login-form/LoginForm.tsx

```diff
@@ -1,39 +1,118 @@
+import { useState } from 'react';
 import TextField from "@mui/material/TextField";
 import Button from "@mui/material/Button";
+import Checkbox from '@mui/material/Checkbox';
 import { FunctionComponent } from "react";
 import useLoginFormState from "./use-login-form-state";
+import { Typography } from "@mui/material";
+import Box from "@mui/material/Box";
+import Link from '@mui/material/Link';
+import ucabLogo from "../images/ucabLogo.webp";
+import "../../../styles/index.css";
+import { TextFieldTypography, 
+  UcabLogoStyles, 
+  ImageBoxStyles,
+  FormBottomTypographyStyles,
+  TextFieldStyles, 
+  ButtonStyles,
+  CheckBoxStyles,
+  QuestionBoxStylesFunct,
+  LinkBoxStylesFunct,
+  RedirectBoxStyles,
+  FormBoxStylesFunct,
+  FormBottomStylesFunct,
+  InsideFormBottomStylesFunct,
+  ForgotPasswordStylesFunct,
+
+} from "./LoginFormStyles.tsx";
 
 const LoginForm: FunctionComponent<Props> = ({ disabled, onSubmit }) => {
   const { email, password, onChangeEmail, onChangePassword } =
     useLoginFormState();
+  
+    const [keepLogged, setKeepLogged] = useState(false);
+    
+    const handleKeepLoggedChange = () => {
+      setKeepLogged(!keepLogged); 
+    };
+
+    const {QuestionBoxStyles} = QuestionBoxStylesFunct();
+    const {LinkBoxStyles} = LinkBoxStylesFunct();
+    const {FormBoxStyles} = FormBoxStylesFunct();
+    const {FormBottomStyles} = FormBottomStylesFunct();
+    const {InsideFormBottomStyles} = InsideFormBottomStylesFunct();
+    const {ForgotPasswordStyles} = ForgotPasswordStylesFunct();
```

Comment:
Los hooks se estan usando relativamente bien, no es tan terrible como pensaba en mis comentarios de arriba por que si se estan usando como hooks, pero debes cambiar el nombre de ellos

```suggestion
    const questionBoxStyles = useQuestionBoxStyles();
    const linkBoxStyles = useLinkBoxStyles();
    const formBoxStyles = useFormBoxStyles();
    const formBottomStyles = useFormBottomStyles();
    const insideFormBottomStyles = useInsideFormBottomStyles();
    const forgotPasswordStyles = useForgotPasswordStyles();
```

Tambien pls, instala prettier para mejorar el espaciado en el codigo